fastify.js

```diff
@@ -917,6 +914,24 @@ function fastify (options) {
     this[kGenReqId] = reqIdGenFactory(this[kOptions].requestIdHeader, func)
     return this
   }
+
+  function acceptHTTPMethod (method, { hasBody = false } = {}) {
```

Comment:
accept is kind of confusing

we have in the HTTP Spec the accept header and the MethodNotSupported status code. 

Maybe simple call id addHTTPMethod?1

Reply:
No strong opinion on this

I chose `accept` over `add` because the former "unlock" an `node:http.METHODS`, while `add` suggest me that I may add whatever HTTP method such `FOO`. I'm overthinking.

Reply:
I agree with `addHttpMethod`. I think we want to allow the user to add whatever crazy methods they can come up with, if possible.